### ISSUE TITLE
NationalMap - GA services update Jul - Aug 2021

### ISSUE THREAD

**Issue by AnaBelgun** [TerriaJS/nationalmap#1037]
Here is a large update on web services hosted by GA. Some of the changes require update to curated National Datasets section of the NationalMap catalogue. 

The checked items can be verified at
https://nationalmap-2021-09-03.nm.terria.io

- [x] Transport - Rail (group)

- [x] Foundation Rail Infrastructure - new service url https://services.ga.gov.au/gis/rest/services/Foundation_Rail_Infrastructure/MapServer 
“Railways” – rename to “Railway Lines” - point to new service
“Railway Stations” – name ok - point to new service
“Railway Bridges” and “Railway Crossing Lines” - leave as is

- [x] Foundation Facilities - new data service to be added under National Datasets - Habitation - Foundation Facilities Points 
https://services.ga.gov.au/gis/rest/services/Foundation_Facilities_Points/MapServer

- [x] In Energy > Electricity Infrastructure > Transmission
“Transmission Lines” – rename to “Electricity Transmission Lines”
“Substations” – rename to “Transmission Substations”

- [x] In Energy > Electricity Infrastructure > Generation
“All Power Stations” – rename to “Power Stations” (these are not _all_ power stations, only the major ones)

- [x] Renewable Energy - add new sub group “Renewable Energy Capacity Factor Maps” https://services.ga.gov.au/gis/rest/services/RenewableEnergyCapacityFactorMaps/MapServer to Renewable Energy folder. 

- [x] Resources - create a new Group under National Datasets and include: 
  - [x] Tennant Creek – Mt Isa IOCG Mineral Potential Assessment, https://services.ga.gov.au/gis/tisa-iocg-min-pot-asmnt/ows
         **The given URL does not work. Use this instead:** https://services.ga.gov.au/gis/rest/services/AusLAMP_TISA_Stations/MapServer
  - [x] North Australian Craton IOCG Mineral Potential Assessment -  https://services.ga.gov.au/gis/nac-iocg-min-pot-assmnt/ows
  - [x] Mineral Blocks - https://services.ga.gov.au/gis/rest/services/OMA_1994_Mineral_Blocks/MapServer
  - [x] Petroleum Blocks - https://services.ga.gov.au/gis/rest/services/OPGGSA_2006_Petroleum_Blocks/MapServer
  - [x] 2021 Offshore Petroleum Acreage - https://services.ga.gov.au/gis/rest/services/2021OffshorePetroleumAcreageReleaseAreas/MapServer
  - [x] Critical Minerals initiative - https://services.ga.gov.au/gis/critical-minerals/wms

- [x] Marine and Oceans
- Bathymetry - add World Bathymetry, Imagery and Hillshade, http://services.ga.gov.au/gis/rest/services/World_Bathymetry_Imagery/MapServer
- add Coastal Waters Act 1970& 1980 -  https://services.ga.gov.au/gis/services/CW_1970_1980/MapServer/WMSServer?request=GetCapabilities&service=WMS
- add Sea & Submerged Lands Act 1973 - https://services.ga.gov.au/gis/rest/services/SSLA_1973/MapServer 

- [x] Resources (Previously in Environment)
- add Environmentally Sustainable Sites For Carbon Dioxide Injection - https://services.ga.gov.au/gis/rest/services/GEODISC_EnvironmentallySustainableSitesForCarbonDioxideInjection/MapServer
- add Advanced CO2 geological storage sites - https://services.ga.gov.au/gis/rest/services/AdvancedCO2GeologicalStorageSites2030/MapServer
- add Potential locations for underground Hydrogen storage - https://services.ga.gov.au/gis/rest/services/PotentialLocationsForUndergroundLargeScaleHydrogenStorage/MapServer
- add Offshore Petroleum and Greenhouse Gas Storage Act 2006 - https://services.ga.gov.au/gis/rest/services/OPGGSA_2006_Areas/MapServer 

Decommissioned Services (with replacements, where applicable)

- [x] Australian Exposure Information and NEXIS Building Exposure (CANNOT FIND THESE IN CATALOGUE)
Equivalent replacement service (now Geoserver):
WMS https://services-em.ga.gov.au/geoserver/exposure/ows?service=wms&version=1.3.0&request=GetCapabilities 
WFS   https://services-em.ga.gov.au/geoserver/exposure/ows?service=wfs&version=2.0.0&request=GetCapabilities

  It is located at https://nationalmap-2021-09-03.nm.terria.io/#share=s-tl5iyEfsNHqWE9SfedQhDBYQsve

- [x] old versions of Offshore Minerals Act 1994 (OMA)  and Offshore Petroleum and Greenhouse Gas Storage Act 2006 (OPGGSA) will all be turned off, so should be removed from the catalogue
Under Boundaries - Other Areas

- [x] 2021 Potential Acreage Release Areas (https://services.ga.gov.au/gis/services/2021PotentialAcreageReleaseAreas/MapServer/WMSServer). The replacement service with existing data is located at https://services.ga.gov.au/gis/services/2021OffshorePetroleumAcreageReleaseAreas/MapServer/WMSServer (see data under Resources)
- [x] Great Artesian Basin Water Table Elevation (https://services.ga.gov.au/gis/rest/services/Great_Artesian_Basin_Water_Table_Elevation/MapServer). Dataset is available in the more recently published “Great Artesian Basin Digital Data” web service at https://services.ga.gov.au/gis/great-artesian-basin/ows

  It is located at https://nationalmap-2021-09-03.nm.terria.io/#share=s-lxHPvDiwkDoa3nrkVjoMJhMW7vV

- [x] Developing Northern Australia White Paper Boundary June 2015 **(Has never been added)**(https://services.ga.gov.au/gis/rest/services/Developing_Northern_Australia_White_Paper_Boundary_June_2015/MapServer) to be decommissioned on 09/09/2021. There is no replacement service.

**Comment by AnaBelgun** [comment]
Update 30 Jun: 
OMA as short for Offshore Minerals Act 1994  and OPGGSA is short for Offshore Petroleum and Greenhouse Gas Storage Act 2006 (both are Commonwealth legislations). These should be under maritime boundaries datasets.

**Comment by mwu2018** [comment]
Proposed group structure for `Foundation Facilities Points` datasets:
![image](https://user-images.githubusercontent.com/41275384/124074660-2ddf9680-da87-11eb-89a3-1b5a4528cf16.png)

**Comment by AnaBelgun** [comment]
If you already put them under Social and Economic lets leave them there for now.

**Comment by wibily** [comment]
ready for Ana to check

**Comment by AnaBelgun** [comment]
To update: 

I suggest that the 4 new services listed to be placed under Environment, should instead be placed under the new Resources group. I have highlighted the ones I mean in the email chain below.

Also the replacement service for the now decommissioned Great Artesian Basin Water Table Elevation could potentially be added under the Water>Ground Water group.

Also if it helps I think I found what the ‘Australian Exposure Information and NEXIS building Exposure dataset’ was referring to under the decommissioned list – I think it should be the ‘Industrial Building Exposure’ dataset, although I too couldn’t find where it actually sits in the National Datasets structure (I only found it through a search of the catalogue).

The rest of the groupings make good sense to me.
